scipy/ndimage/tests/test_filters.py

```diff
@@ -20,6 +20,29 @@ def test_ticket_701():
     assert_equal(res, res2)
 
 
+def test_gh_5430():
```

Comment:
Since this means to fix gh-5430, and the latter includes a short piece of code which shows the problem, this code piece should be included as a regression test.